apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue

```diff
@@ -72,9 +72,23 @@ const state = reactive({
         end: dayjs.utc(state.settings?.date_range?.end).format(DATE_FORMAT),
     })),
     // TODO: remove sampleData
+    legends: computed<Legend[]|undefined>(() => {
+        if (props.widgetConfigId === countOfPassAndFailFindingsWidgetConfig.widget_config_id) {
+            return [
+                { name: 'pass', label: 'Pass', color: green[500] },
+                { name: 'fail', label: 'Fail', color: red[400] },
+            ];
+        } if (props.widgetConfigId === countOfFailFindingsWidgetConfig.widget_config_id) {
+            return [
+                { name: 'fail', label: 'Fail', color: red[400] },
+            ];
+        }
+        return undefined;
+    }),
```

Comment:
Note: The only difference between the two widgets is that one has 'pass' data and the other does not.

Reply:
For others' information..

![image](https://github.com/cloudforet-io/console/assets/26986739/d8acc9e4-47b3-4270-aee7-465afd03952f)